content/src/content/jcr_root/apps/core/wcm/components/embed/v1/embed/embeddable/youtube/youtube.html

```diff
@@ -13,13 +13,28 @@
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License.
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
-<iframe data-sly-use.youTube="com.adobe.cq.wcm.core.components.models.embeddable.YouTube"
-        data-sly-test="${!youtube.empty}"
-        data-cmp-data-layer="${youtube.data.json}"
-        width="${youTube.iFrameWidth || '100%' @ context='scriptString'}"
-        height="${youTube.iFrameHeight || 390 @ context='scriptString'}"
-        src="${youTube.iFrameSrc}"
-        frameborder="0"
-        allowfullscreen
-        allow="autoplay; fullscreen"
-        aria-label="${'YouTube Video' @ i18n}"></iframe>
+
+<sly data-sly-use.youTube="com.adobe.cq.wcm.core.components.models.embeddable.YouTube">
+  <sly data-sly-test="${!youtube.empty}">
+    <iframe data-sly-test="${!youtube.layout || youtube.layout=='fixed'}"
```

Comment:
I'd rather we split the two templates (fixed/responsive) into separate HTL template files so that they can be more easily overridden.